packages/airnode-validator/src/config/config.ts

```diff
@@ -16,7 +16,7 @@ export const triggerSchema = z
 export const triggersSchema = z
   .object({
     rrp: z.array(triggerSchema),
-    http: z.array(triggerSchema).optional(),
```

Comment:
Yeah, I am working on another validator issue and I think there are multiple fields which should not be optional... I'll be creating similar commits to what you are doing now.